sumologic/config/user.properties

```diff
@@ -0,0 +1,4 @@
+name = {{cfg.collector_name}}
```

Comment:
The docs say the name is not changable after install.. will be interesting to see if that's the case.

Reply:
I remember this from using Sumo Logic a few years back, we should use the hostname of the instance or perhaps a randomly generated GUID

Reply:
It's mostly useful to tell what collectors we in each env. The hostname is sent with all events so we don't really ever have to go back and cross-check collectors.